app/code/Magento/Sales/Model/Order/ShipmentDocumentFactory.php

```diff
@@ -88,6 +99,8 @@ public function create(
             $shipmentItems
         );
 
+        $this->extensionAttributesProcessor->execute($shipment, $arguments);
```

Comment:
We could remove `@SuppressWarnings(PHPMD.UnusedFormalParameter)` from dock block

Reply:
I think we don't need delete it because parameter $packages isn't used